include/sensors/core/humidity_sensor.hpp

```diff
@@ -0,0 +1,54 @@
+#pragma once
+
+#include "can/core/ids.hpp"
+
+/*
+ * Configurations and register information for the TI HDC2080 sensor
```

Comment:
this is a good comment but it's the only point in any of this code where you're reminded what this sensor is. for sensor or remote ic bindings, I often find it helpful to have files like this, that store values related to not just humidity or enviro sensors but _this specific chip_, be named after the chip and maybe what it does - like `enviro_sensor_hdc2080.hpp` and the namespace be like `hdc2080`